src/rails_new.rs

```diff
@@ -0,0 +1,31 @@
+use clap::Parser;
+
+#[derive(Parser)]
+#[command(version, about, long_about = None)]
+pub struct Cli {
+    #[clap(trailing_var_arg = true, required = true)]
+    pub args: Vec<String>,
```

Comment:
If you add a comment above args then it'll be used when the usage string is printed.

E.g.:

```patch
diff --git a/src/rails_new.rs b/src/rails_new.rs
index d61a5b4..626dba4 100644
--- a/src/rails_new.rs
+++ b/src/rails_new.rs
@@ -4,6 +4,7 @@ use clap::Parser;
 #[command(version, about, long_about = None)]
 pub struct Cli {
     #[clap(trailing_var_arg = true, required = true)]
+    /// arguments passed to `rails new`
     pub args: Vec<String>,
     #[clap(long, short = 'u', default_value = "3.2.3")]
     pub ruby_version: Option<String>,
```

improves the message from:

```
A CLI tool to generate a new Rails project

Usage: rails-new [OPTIONS] <ARGS>...

Arguments:
  <ARGS>...  

Options:
  -u, --ruby-version <RUBY_VERSION>    [default: 3.2.3]
  -r, --rails-version <RAILS_VERSION>  [default: 7.1.3]
  -h, --help                           Print help
  -V, --version                        Print version
```

to

```
A CLI tool to generate a new Rails project

Usage: rails-new [OPTIONS] <ARGS>...

Arguments:
  <ARGS>...  arguments passed to `rails new`

Options:
  -u, --ruby-version <RUBY_VERSION>    [default: 3.2.3]
  -r, --rails-version <RAILS_VERSION>  [default: 7.1.3]
  -h, --help                           Print help
  -V, --version                        Print version

```

Reply:
Amazing. Applied